transport/http/outbound.go

```diff
@@ -111,30 +115,65 @@ func (o outbound) Call(ctx context.Context, req *transport.Request) (*transport.
 	deadline, _ := ctx.Deadline()
 	ttl := deadline.Sub(start)
 
-	request, err := http.NewRequest("POST", o.URL, req.Body)
+	req, err := http.NewRequest("POST", o.URL, treq.Body)
 	if err != nil {
 		return nil, err
 	}
 
-	request.Header = applicationHeaders.ToHTTPHeaders(req.Headers, nil)
+	// Apply HTTP Context headers for tracing and baggage carried by tracing.
```

Comment:
👍 - that's exactly how I did OpenTracing in all TChannel impls, the main code just calls into helper functions which at all consolidated into a single file `tracing.go`